src/parsing/binast-deserializer.cc

```diff
@@ -44,13 +41,13 @@ AstNode* BinAstDeserializer::DeserializeAst(ByteArray serialized_ast) {
   int offset = 0;
   auto string_table_result = DeserializeStringTable(uncompressed_byte_array.get(), offset);
   offset = string_table_result.new_offset;
-  auto result = DeserializeAstNode(uncompressed_byte_array.get(), offset);
+  auto result = DeserializeAstNode(uncompressed_byte_array.get(), offset, true);
```

Comment:
Might be nice to store this as a named variable so we know what the param name is. Just a readability nit.